simulations/vip-123/vip-123/simulations.ts

```diff
@@ -0,0 +1,295 @@
+import { expect } from "chai";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { expectEvents, setMaxStalePeriodInChainlinkOracle, setMaxStalePeriodInOracle } from "../../../src/utils";
+import { forking, testVip } from "../../../src/vip-framework";
+import { vip123 } from "../../../vips/vip-123/vip-123";
+import CHAINLINK_ORACLE_ABI from "./abi/chainlinkOracle.json";
+import COMPTROLLER_ABI from "./abi/comptroller.json";
+import PRICE_ORACLE_ABI from "./abi/priceOracle.json";
+import RESILIENT_ORACLE_ABI from "./abi/resilientOracle.json";
+
+const COMPTROLLER = "0xfd36e2c2a6789db23113685031d7f16329158384";
+const CHAINLINK_ORACLE = "0x1B2103441A0A108daD8848D8F5d790e4D402921F";
+const PRICE_ORACLE = "0x7FabdD617200C9CB4dcf3dd2C41273e60552068A";
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+
+interface vTokenConfig {
```

Comment:
Even though `vToken` is usually lowercased, I don't think it's a valid exception for the types naming convention

```suggestion
interface VTokenConfig {
```